tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S

```diff
@@ -25,35 +25,35 @@ _start0:
 	csrrw x14, 2816, x7
```

Comment:
You say that the csr_access test is updated. Could you give a quick recap on what are updated?

Reply:
This was updated: https://github.com/openhwgroup/cv32e40x/pull/957

And... actually, in the 3 months since I PRed this, there was another update to that file. I will have to re-run the generation and run the test again.
Let's just leave this for now.

For completeness:
After that Yaml change, then the regen script described in https://github.com/openhwgroup/cv32e40x-dv/pull/37/files#diff-43e71d36024fbe1d133827162b04bf6e64b864e979725f7c9f09f96562ccc09e was run.
That auto-generated the assembly file to match our most recent CSR definitions.

Reply:
Okei, I see. Thank you. Okei, then we leave it for now.